change/@fluentui-react-color-picker-4fba51df-d53c-4e1e-9f55-67107d298f67.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat: release stable",
+  "packageName": "@fluentui/react-color-picker",
+  "email": "email not defined",
```

Comment:
can you please configure your git accordingly ?

Reply:
That was from WSL. I'll fix it